addons/mail/static/src/core/common/message_model.js

```diff
@@ -40,6 +49,23 @@ export class Message extends Record {
     attachments = Record.many("Attachment", { inverse: "message" });
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
+    bodyMarkdown = Record.attr("", {
+        compute() {
+            if (!this.isBodyEmpty) {
+                if (!this.store.markdownLoaded) {
+                    loadMarkdownAssets.loadMarkdown().then(() => this.store.markdownLoaded = true);
+                    return this.body;
+                }
+                const htmlDoc = parser.parseFromString(this.body, "text/html");
+                const markdownElements = htmlDoc.querySelectorAll("odoo-markdown");
```

Comment:
This could be checked before, if there's no `odoo-markdown` we don't need to load the lib.